server/api/teams/resources.py

```diff
@@ -293,8 +319,37 @@ def get(self):
             500:
                 description: Internal Server Error
         """
+
+        filters = {}
+        filters["team_name_filter"] = request.args.get("team_name")
+        try:
+            member_filter = request.args.get("member")
+            filters["member_filter"] = int(member_filter) if member_filter else None
+        except ValueError:
+            pass
+
+        try:
+            manager_filter = request.args.get("manager")
+            filters["manager_filter"] = int(manager_filter) if manager_filter else None
```

Comment:
Would it cause an error if we don't convert to int? When I pass a string instead of returning 0 results, it didn't filter